packages/fast-component-explorer/app/explorer.tsx

```diff
@@ -437,6 +545,39 @@ class Explorer extends Foundation<ExplorerHandledProps, {}, ExplorerState> {
     private get backgrounds(): typeof DarkModeBackgrounds | typeof LightModeBackgrounds {
         return DarkModeBackgrounds;
     }
+
+    private getNeutralPalette(colorSource: string): string[] {
+        const color: ColorRGBA64 | null = parseColor(colorSource);
+        if (color !== null) {
+            const hslColor: ColorHSL = rgbToHSL(color);
+            const augmentedHSLColor: ColorHSL | null = ColorHSL.fromObject({
+                h: hslColor.h,
+                s: hslColor.s,
+                l: 0.5,
+            });
+
+            if (augmentedHSLColor !== null) {
+                return createColorPalette(hslToRGB(augmentedHSLColor));
+            }
+        }
+        return DesignSystemDefaults.neutralPalette;
+    }
+
+    private handleUpdateTheme = (): void => {
+        this.setState({
+            viewConfig: merge({}, this.state.viewConfig, {
+                theme:
```

Comment:
I would remove theme from the viewConfig and just have it as part of state, so `this.state.theme` since it's not part of the design system and yet if you add it to viewConfig, it will be added to the previews <DesignSystemProvider /> which is incorrect